code/modules/mob/living/simple_mob/subtypes/mechanical/mecha/mecha.dm

```diff
@@ -124,7 +124,7 @@
 	sleep(1 SECOND)
 	cut_overlay(deflect_image)
 	qdel(deflect_image)
-//	flick_overlay_view(deflect_image, src, duration = 1 SECOND, gc_after = TRUE)
+//	flick_overlay_view(deflect_image, duration = 1 SECOND)
```

Comment:
```suggestion
```

Reply:
Delete code, not commenting out,
unless the commented out code helps improve readabilty